client/src/components/Homepage-UI/Homepage.css

```diff
@@ -0,0 +1,209 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css");
+@import url("https://fonts.googleapis.com/css?family=Montserrat:700");
```

Comment:
Please change the font family, we will use Poppins for heading and open sans for paragraphs